docs/src/components/pricing/PricingTable.tsx

```diff
@@ -107,7 +108,7 @@ export function PlanPrice(props: PlanPriceProps) {
           Billed annually at $180/dev.
         </Typography>
         <Typography variant="body2" color="text.secondary" sx={{ mb: 3 }}>
-          Price capped at 10 developers.
+          Licenses beyond 10 devs are at $0.
```

Comment:
To clear the confusion in https://mui.zendesk.com/agent/tickets/6428

Reply:
I think it would phrase it more like this:

```suggestion
          There's no additional charge for more than 10 developers.
```

Reply:
The text wraps on two lines. Ideally, it would stay in a single line to be quick to read.

Reply:
Hmm, that could potentially lead to someone thinking that the license is free as long as they have 11 or more devs. 😅 Maybe something like this to keep the character count down?

```suggestion
          No additional fee beyond 10 devs.
```